apps/spruce/src/analytics/task/useTaskAnalytics.ts

```diff
@@ -74,6 +74,7 @@ export const useTaskAnalytics = () => {
 
   const [execution] = useQueryParam(RequiredQueryParams.Execution, 0);
   const { data: eventData } = useQuery<TaskQuery, TaskQueryVariables>(TASK, {
+    skip: !taskId || execution === undefined,
```

Comment:
Does `execution === undefined` ever evaluate to true? It seems like the variable has a default of 0 from `useQueryParam`